requirements.txt

```diff
@@ -0,0 +1,6 @@
+numpy
+numba
+pymc3
+scipy
+sklearn
```

Comment:
should be `scikit-learn`, i believe. some kind stranger keeps a redirect at `sklearn`. Check out https://pypi.org/project/sklearn/. Its a hoot.

Reply:
Woops